.github/workflows/ci.yml

```diff
@@ -417,6 +421,11 @@ jobs:
           secret_vault_roleId: ${{ secrets.VAULT_ROLE_ID }}
           secret_vault_secretId: ${{ secrets.VAULT_SECRET_ID }}
 
+  optimize-unit-tests:
+    name: Unit Tests
+    uses: ./.github/workflows/optimize-unit-tests.yml
+    secrets: inherit
```

Comment:
❌ Please take a look at the other jobs. We need to restrict the execution via the path filtering, otherwise it is always run.

Furthermore, right now it is not considered in the check results. That means even it is running it doesn't matter as the result it just ignored.

Example from java checks
>   java-checks:
>     if: needs.detect-changes.outputs.java-code-changes == 'true'
>     needs: [detect-changes]

Reply:
there is a path filtering in that workflow itself so I thought it makes no sense to duplicate it. Does it make sense or should I still add another path filter nayway?

Reply:
I think it should follow the pattern described here:
https://github.com/camunda/camunda/wiki/CI-&-Automation#implementation

Reply:
Yes please align with and follow the pattern that Omran linked to have a common approach everywhere